Include/code.h

```diff
@@ -49,6 +51,14 @@ typedef struct {
        Type is a void* to keep the format private in codeobject.c to force
        people to go through the proper APIs. */
     void *co_extra;
+
+    /* Per opcodes just-in-time cache */
+    // To reduce cache size, we use indirect mapping from opcode index
+    // to cache object: co_opcache[co_opcache_map[next_instr/sizeof(_Py_CODEUNIT)]]
+    unsigned char *co_opcache_map;
+    _PyOpcache *co_opcache;
+    int32_t co_opcache_flag;  // used to determine when create cache
```

Comment:
(1) It looks like this is a usage count, rather than a flag.  Maybe call it something like co_run_count?
(2) I don't think you need a full 32 bit int -- which means that it could fit into the same word as the following field.

Reply:
Yes -- currently.  I don't think current algorithm to determine when enable cache
is stable, it can be reconsidered several times.  So I don't think we need name represents
concrete usage of the variable for now.